modules/components/pages/redpanda/about.adoc

```diff
@@ -4,9 +4,12 @@ component_type_dropdown::[]
 
 // tag::single-source[]
 
-// © 2024 Redpanda Data Inc.
+The Redpanda Connect configuration service allows you to:
 
-You can configure Redpanda Connect to send logs and status updates to topics on a Redpanda cluster, in addition to the xref:components:logger/about.adoc[default logger]. The `redpanda` namespace contains the configuration of this service, which uses the following default values:
+- Configure Redpanda cluster credentials in a single configuration block, which is referenced by multiple components in data pipeline. For more information, see the <<pipeline-example,Pipeline example>>.
```

Comment:
Is this accurate? Also, we seem to have lost the `rack_id` field from the previous version, we that intentional?

Reply:
Yeah, the rack id is only relevant for inputs and so the field has been moved out (it's deprecated so existing users won't end up with broken configs).